tasks/__init__.py

```diff
@@ -50,8 +50,7 @@
     reset,
     tidy_all,
 )
-from .show_linters_issues import show_linters_issues
-from .test import (
+from .go_test import (
```

Comment:
I needed to do that to prevent name collision while trying to import it on unit tests